src/main/java/org/junitpioneer/jupiter/StdIoStream.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.io.OutputStream;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+
+abstract class StdIoStream extends OutputStream {
```

Comment:
Isn't it more of a `StdOutStream`, `StdOutputStream`, `StdPrintStream`, ... as it only extends `OutputStream` and not `InputStream`?

Reply:
Yeah, 100%. Thank you!